src/shogun/classifier/mkl/MKL.h

```diff
@@ -95,6 +95,8 @@ class CMKL : public CSVM
 		 */
 		virtual ~CMKL();
 
+		CSGObject* clone();
```

Comment:
Writing `virtual` is just eye candy, it's virtual if it's virtual in base. The only specifier that would make an actual difference here is `override`. If you want the `virtual` here for code style, I can add it.